hs_core/views/__init__.py

```diff
@@ -1155,9 +1156,28 @@ def update(self, group_to_update, request):
 @processor_for('my-resources')
 @login_required
 def my_resources(request, page):
-
+    """
+    TODO parameterize and give user control of listings per page, saved in user preferences
+    Template processor for my-resources.html resources listing for user
+    :param request: WSGI GET request for endpoint
+    :param page: Mezzanine middleware in process_view sends <Page: My Resources>
+    :return: html template render with page of resources
+    """
     resource_collection = get_my_resources_list(request)
-    context = {'collection': resource_collection}
+    page = request.GET.get('page')
+    # TODO investigate iRODS performance issue; listing caps out at 10 resources per page, otherwise very slow loading
+    paginator = Paginator(resource_collection, 10)
```

Comment:
Let me know if I am wrong. The benefit we are getting using the pagination is that the user doesn't have to scroll through a long list and the page loading time is quicker as the template engine now has to process only 10 records.

Reply:
It became a higher priority, due to 30+ second load times for CZO data, due to high number of complex resources, and your statement is correct